backend/sequelize/models/service.model.js

```diff
@@ -0,0 +1,30 @@
+const { DataTypes } = require("sequelize");
+
+module.exports = (db) => {
+  db.define("service", {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    category: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    estimateUnit: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    priceEstimate: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    description: {
+      type: DataTypes.STRING,
+      allowNull: false,

```

Comment:
(Just in case) Description might be null, or we might want it for sure, both works.

Reply:
Making it not Null just to make implementation easier. Don't need to check if it exists later